README.md

```diff
@@ -44,73 +29,49 @@ Each display will have have 5 pins that will all go to/share the same output pin
     RST -> G18
     VCC - >5V/VCC 
     GND -> GND
-
-Each screen will have its own unique `CS` pin on the ESP32. The CS pin numbers for each screen, from left to right are
-
     Screen1 CS -> G13
     Screen2 CS -> G33
     Screen3 CS -> G32 
     Screen4 CS -> G25
     Screen5 CS -> G21
    
-   Lastly, wiring up a single pushbutton between `VCC/5V` and `G26`
-
+   Lastly, three pushbuttons between `VCC/5V` and `G14,G26,G27.`
 Diagram can be seen below: 
 <img src="references/wiringDiag.png" alt="Wiring Diagram">
 
 
 ### 2. Dev Environment Setup/Project Config
 **IDE Setup**
 
-While built on Arduino, the codebase for this project has been built using the  [Platform IO IDE](https://platformio.org/), which allows you to compile/run Arduino code in VSC, as well as quite a few other creature comfort items.
+While built on Arduino, the codebase for this project has been built using the  [Platform IO IDE](https://platformio.org/), which allows you to compile/run Arduino code in VS as well as quite a few other creature comfort items.
 Once you have platform.io installed and configured in VSC, you'll want to select "Open Project", and select & open the main "Info-Orbs" directory.
 
 **Project Configuration**
 (all of these items apply only to bretts code on the main branch, the nightly config is slightly different)
 
-Before compiling/flashing, you'll want to adjust a few parameters in src/main.cpp:
+Before compiling/flashing, you'll need to naviagte into `Info-Orbs` >>` lib `>> `config` directory and change the file name from `config.h.template` to `config.h` **THIS STEP IS CRITICAL AND YOUR CODE WILL NOT COMPILE IF YOU DONT CHNAGE THIS FILE NAME**
```

Comment:
copying makes more send than renaming because when you next do a "git pull" it will show the missing file as a local change.